typed-racket-lib/typed-racket/types/abbrev.rkt

```diff
@@ -12,12 +12,18 @@
          racket/function
 
          (prefix-in c: (contract-req))
-         (rename-in (rep type-rep prop-rep object-rep values-rep)
+         (rename-in (rep rep-utils type-rep prop-rep object-rep values-rep)
                     [make-Base make-Base*])
-         (types union numeric-tower prefab)
+         (types numeric-tower prefab)
          ;; Using this form so all-from-out works
          "base-abbrev.rkt" "match-expanders.rkt"
 
+         ;; signature env req here is so it is statically required by
+         ;; the code loaded during typechecking, otherwise we get
+         ;; a `reference to a module that is not available` error
+         ;; from references generated by init-envs
+         (env signature-env)
```

Comment:
`init-envs` produces syntax which refers to `register-signature!`, which is defined in `signature-env.rkt`. Without this I get those "module not available" errors from the references to `register-signature!` since `signature-env.rkt` is not loaded.